packages/driver/test/cypress/fixtures/app.js

```diff
@@ -1 +1 @@
-{foo: "bar"}
+{ 'bar' }
```

Comment:
why such weird javacript string here?

Reply:
I'm not sure what that's about. I think eslint keeps auto-formatting it for some reason.